### DISCUSSION TITLE
New Exploit maybe? *uses illuminate secure browser app

### DISCUSSION THREAD

**Discussion by ghost** [3kh0/ext-remover#1357] [Webview JOIN TITANIUM NETWORK TO SHARE PLEASE]
FOR PEOPLE WHO HAVE ILLUMINATE SECURE BROWSER IN THEIR APPS WHEN THEY JUST HAVE STARTED THEIR COMPUTER. 
Step 1. Sign out
Step 2. Go to apps on the left down corner of the screen and then click ILLUMINATE SECURE BROWSER 
Step 3. IDK IF YALL HAVE IN THE NEXT PAGE THAT POPS UP BUT IT SHOULD SHOW LIKE SELECT A SCHOOL AND THEN YOU JUST PUT IN HENRY COUNTY PUBLIC SCHOOL. 
Step 4. IMPORTANT now what you want to do is just click the Renaissance logo saying “seeing every student” and click it. It SHOULD be on the right bottom corner and then it should say login to your account.
Step 5. now go down to sign in options and click it and then go to sign in with GitHub. 
Step 6. Click privacy on the bottom then go all the way down from the article and then click ask the community
Step 7. Delete everything in the search bar of github (u should be in github) and then type in [www.google.com](http://www.google.com/) and try to find a link to google.
Step 8. After you find that you'll be able to search anything up but there are some things like I couldn’t sign into github for some reason but I could do everything else for me though.

**Comment by ghost** [comment]
LET ME KNOW

**Comment by Exploiter-org** [comment]
I don't got illuminate

**Comment by ghost** [comment]
*FOR PEOPLE WHO HAVE ILLUMINATE*
